### PR TITLE
Don't make funnels horizontal

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

The results have come in and the people didn't want horizontal funnels! 

Screen Shot 2022-02-14 at 3 32 29 PM

Thus, let's remove that FF and clean up our code accordingly.

## How did you test this code?

Nothing breaks.

### CODE REVIEW

File: frontend/src/scenes/funnels/FunnelBarGraph.tsx [hunk 1]
```diff
@@ -308,12 +308,7 @@ export function FunnelBarGraph({ color = 'white' }: { color?: string }): JSX.Ele
     return (
         <div
             data-attr="funnel-bar-graph"
-            className={clsx(
-                'funnel-bar-graph',
-                FunnelLayout.horizontal,
```
Comment (paolodamico) [inline frontend/src/scenes/funnels/FunnelBarGraph.tsx:313]: We should probably get rid of this `FunnelLayout` type too (and related stuff)

**Review by paolodamico — COMMENTED** [review]
Before merging, I wonder if we should take a closer look at the experiment results (CC @neilkakkar too for input).

- Total goal metric looks very close to control group, and looks like the split of users was roughly [the same](https://app.posthog.com/insights/EDEPi8VY/edit#filters=%7B%22insight%22%3A%22TRENDS%22%2C%22interval%22%3A%22week%22%2C%22display%22%3A%22ActionsLineGraph%22%2C%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22insight%20analyzed%22%2C%22name%22%3A%22insight%20analyzed%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22properties%22%3A%5B%7B%22key%22%3A%22%24feature%2F5730-funnel-horizontal-ui%22%2C%22value%22%3A%5B%22control%22%5D%2C%22operator%22%3A%22exact%22%2C%22type%22%3A%22event%22%7D%5D%2C%22math%22%3A%22dau%22%7D%2C%7B%22id%22%3A%22insight%20analyzed%22%2C%22name%22%3A%22insight%20analyzed%22%2C%22type%22%3A%22events%22%2C%22order%22%3A1%2C%22properties%22%3A%5B%7B%22key%22%3A%22%24feature%2F5730-funnel-horizontal-ui%22%2C%22value%22%3A%5B%22test%22%5D%2C%22operator%22%3A%22exact%22%2C%22type%22%3A%22event%22%7D%5D%2C%22math%22%3A%22dau%22%7D%5D%2C%22properties%22%3A%5B%5D%2C%22filter_test_accounts%22%3Atrue%2C%22new_entity%22%3A%5B%5D%2C%22date_from%22%3A%22-90d%22%2C%22date_to%22%3Anull%7D).
- Was trying to understand the threshold we used, as improvement is close to 3%, but it doesn't look like the app tells you that on the summary screen.
- The target metric just looks like insight analyzed, but I think we should filter by `insight = FUNNELS` and see if the experiment results still hold.

**Comment by neilkakkar** [comment]
[It's closer to 20% lift for insight=FUNNEL](https://app.posthog.com/insights/EDEPi8VY/edit#filters=%7B%22insight%22%3A%22TRENDS%22%2C%22interval%22%3A%22week%22%2C%22display%22%3A%22ActionsLineGraphCumulative%22%2C%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22insight%20analyzed%22%2C%22name%22%3A%22insight%20analyzed%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22properties%22%3A%5B%7B%22key%22%3A%22insight%22%2C%22value%22%3A%5B%22FUNNELS%22%5D%2C%22operator%22%3A%22exact%22%2C%22type%22%3A%22event%22%7D%5D%2C%22math%22%3A%22total%22%7D%2C%7B%22id%22%3A%22insight%20analyzed%22%2C%22name%22%3A%22insight%20analyzed%22%2C%22type%22%3A%22events%22%2C%22order%22%3A1%2C%22properties%22%3A%5B%7B%22key%22%3A%22insight%22%2C%22value%22%3A%5B%22FUNNELS%22%5D%2C%22operator%22%3A%22exact%22%2C%22type%22%3A%22event%22%7D%5D%2C%22math%22%3A%22total%22%7D%5D%2C%22properties%22%3A%5B%5D%2C%22filter_test_accounts%22%3Atrue%2C%22new_entity%22%3A%5B%5D%2C%22date_from%22%3A%22-30d%22%2C%22date_to%22%3Anull%2C%22breakdown%22%3A%22%24feature%2F5730-funnel-horizontal-ui%22%2C%22breakdown_type%22%3A%22event%22%7D)s

[and yep, exposure is roughly the same](https://app.posthog.com/insights/EDEPi8VY/edit?order=count#filters=%7B%22insight%22%3A%22TRENDS%22%2C%22interval%22%3A%22week%22%2C%22display%22%3A%22ActionsLineGraphCumulative%22%2C%22actions%22%3A%5B%5D%2C%22events%22%3A%5B%7B%22id%22%3A%22%24feature_flag_called%22%2C%22name%22%3A%22%24feature_flag_called%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22properties%22%3A%5B%5D%2C%22math%22%3A%22dau%22%7D%5D%2C%22properties%22%3A%5B%7B%22key%22%3A%22%24feature_flag%22%2C%22value%22%3A%5B%225730-funnel-horizontal-ui%22%5D%2C%22operator%22%3A%22exact%22%2C%22type%22%3A%22event%22%7D%5D%2C%22filter_test_accounts%22%3Atrue%2C%22new_entity%22%3A%5B%5D%2C%22date_from%22%3A%22-90d%22%2C%22date_to%22%3Anull%2C%22breakdown%22%3A%22%24feature_flag_response%22%2C%22breakdown_type%22%3A%22event%22%7D).

I think with 10 to 20,000 events, a precision close to 3% is possible.

> Was trying to understand the threshold we used, as improvement is close to 3%, but it doesn't look like the app tells you that on the summary screen.

The threshold doesn't really matter at this stage, does it? All you have is the events data, which in turn determines minimum precision / whether the results are significant, even if they're under the threshold you defined (because, say, experiment has run for longer than its run time, giving you more time to collect data).

... Although, something else does look off. Investigating.

**Comment by neilkakkar** [comment]
There was indeed a bug with exposure calculation! We were double counting daily active users if they came back on another day (sigh).  This should fix it: https://github.com/PostHog/posthog/pull/8600 - but might also switch around results a bit here!

Actually, will *definitely switch around* results a bit here when this goes in^, but had another quick look, and specific to funnel insights, putting the new numbers in, this experiment would still be significant.

File: frontend/src/scenes/funnels/FunnelBarGraph.tsx (hunk 1 above)
Comment (mariusandra) [reply frontend/src/scenes/funnels/FunnelBarGraph.tsx:313]: These are actually for the Left-to-right vs top-to-bottom distinction. Not for the UI.

**Review by mariusandra — APPROVED** [review]
LGTM.

**Comment by mariusandra** [comment]
Apologies for the merge @paolodamico, I missed your "before merging" comment. Any reason to revert?

**Comment by paolodamico** [comment]
No worries @mariusandra, no reason to revert, @neilkakkar validated the conclusions

**Pull request merged**